src/Server/Test/Unit/Repositories/ClaraPayloadsApiTest.cs

```diff
@@ -56,7 +56,7 @@ public void Upload_ShallThrowWithBadPayloadId()
 
             var exception = Assert.Throws<AggregateException>(() =>
```

Comment:
why not make the lamda `async` and use `await` in the method instead of `.Wait()`?

Reply:
Fixed in #64